freestanding/Makefile

```
@@ -1,4 +1,6 @@
-PKG_CONFIG_PATH := $(shell opam config var prefix)/lib/pkgconfig
+ifneq (, $(shell command -v opam))
+	PKG_CONFIG_PATH ?= $(shell opam config var prefix)/lib/pkgconfig
```

```suggestion
	PKG_CONFIG_PATH ?= $(shell opam var prefix)/lib/pkgconfig
```

let's proactively avoid warnings with opam 2.1 (see https://github.com/ocaml/opam/issues/4503)

---

Let's see if CI likes it.